.github/workflows/ci.yml

```diff
@@ -14,12 +14,16 @@ jobs:
 
     - name: Build docker images
       run: |
-        docker build -t exaworks/sdk-base --cache-from exaworks/sdk-base docker/base
+        docker build -t exaworks/sdk-base --cache-from exaworks/sdk-base docker/base/centos8
         docker build -t rp docker/rp
         docker build -t rp_parsl --build-arg BASE_IMAGE=rp docker/Parsl
         docker build -t rp_parsl_swift --build-arg BASE_IMAGE=rp_parsl docker/swift-t
         docker build -t rp_parsl_swift_flux --build-arg BASE_IMAGE=rp_parsl_swift docker/flux
 
+    - name: Build docker base images
+      run: |
+        docker build -t exaworks/sdk-base --cache-from exaworks/sdk-base docker/base/centos7
+
```

Comment:
@SteVwonder should other docker builts should be here? or this part can be kept separately to build other base images? (and default base image is set CentOS8)

Reply:
Yeah, I think it would be good to build the other images on top of the centos 7 base image as well as run the tests below.  I'd recommend checking out github action's matrix strategy: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix.  Leveraging that, you can avoid duplicating a lot of YAML, plus each of the configurations that the matrix strategy produces will run in parallel.  Which means we can test all of our various OS variants in parallel rather than serial.

Reply:
Extremely useful, thank you! Just updated accordingly